src/reference/webhooks/verify.js.txt

```diff
@@ -0,0 +1,8 @@
+const crypto = require('crypto');
```

Comment:
`.txt`?

Reply:
To keep eslint from linting it, comtpalining about unused functions and such

Reply:
You could exclude the file from eslint, which would be cleaner imo, but I will leave this up to you.